### ISSUE TITLE
Checkout: address-i18n.js does not respect the fields priority & label set by woocommerce_billing_fields filter

### ISSUE THREAD

**Issue by des79** [woocommerce/woocommerce#22665]
**Describe the bug**
On checkout page I've my custom fields priority and label (not via po file) but address-i18n.js this settings.

Via woocommerce_billing_fields filter I set, for example, billing_address_1 args and I've this:
`
[billing_address_1] => Array (
  [label] => Indirizzo
  [placeholder] => Via/Piazza
  [required] =>
  [class] => Array(
  [0] => form-row-first
  )
  [autocomplete] => address-line1
  [priority] => 140
  )
`
but method localize_script set wc_address_i18n_params with params taken from get_script_data in class-wc-frontend-scripts.php that get data from method get_country_locale in class-wc-countries.php (line739) and set this:

`
[address_1] => Array(
  [label] => Via e numero
  [placeholder] => Via/Piazza e Numero Civico
  [required] => 1
  [class] => Array(
  [0] => form-row-wide
  [1] => address-field
  )  
  [autocomplete] => address-line1
  [priority] => 50
  )
`
so address-i18n.js order fields based on this args and not with args set via woocommerce_billing_fields filter.
I need to use woocommerce_get_country_locale_base and replace args with data taken via woocommerce_billing_fields filter.

I think it's redundant and is better to use data taken only from woocommerce_billing_fields 

This happen only with WC 3.5.4 and not previous version 3.5.3. I've production server with WC 3.5.3 and stage server with 3.5.4.

**To Reproduce**
1. Modify some fields with custom priority and label via woocommerce_billing_fields filter.
2. Load checkout page, and check if fields it's in the position we are expecting (or if it has the label set )

**Screenshots**
N/A

**Expected behavior**
Correct fields order and label

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

Home URL: https://s.piuchepuoi.it
Site URL: https://s.piuchepuoi.it
WC Version: 3.5.4
Log Directory Writable: ✔
WP Version: 5.0.3
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: ✔
WP Cron: –
Language: it_IT
External object cache: ✔

### Server Environment ###

Server Info: Apache/2.4.6
PHP Version: 7.2.9
PHP Post Max Size: 8 MB
PHP Time Limit: 120
PHP Max Input Vars: 1000
cURL Version: 7.29.0
NSS/3.36

SUHOSIN Installed: –
Max Upload Size: 2 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.5.4
WC Database Prefix: wp_
MaxMind GeoIP Database: ✔
Dimensione totale database: 2825.49MB
Dimensione dati database: 1681.16MB
Dimensione indice database: 1144.33MB
wp_woocommerce_sessions: Dati: 2.52MB + Indice: 0.08MB
wp_woocommerce_api_keys: Dati: 0.02MB + Indice: 0.03MB
wp_woocommerce_attribute_taxonomies: Dati: 0.02MB + Indice: 0.02MB
wp_woocommerce_downloadable_product_permissions: Dati: 0.05MB + Indice: 0.05MB
wp_woocommerce_order_items: Dati: 0.09MB + Indice: 0.05MB
wp_woocommerce_order_itemmeta: Dati: 0.44MB + Indice: 0.44MB
wp_woocommerce_tax_rates: Dati: 0.02MB + Indice: 0.06MB
wp_woocommerce_tax_rate_locations: Dati: 0.02MB + Indice: 0.05MB
wp_woocommerce_shipping_zones: Dati: 0.02MB + Indice: 0.00MB
wp_woocommerce_shipping_zone_locations: Dati: 0.02MB + Indice: 0.03MB
wp_woocommerce_shipping_zone_methods: Dati: 0.02MB + Indice: 0.00MB
wp_woocommerce_payment_tokens: Dati: 0.02MB + Indice: 0.02MB
wp_woocommerce_payment_tokenmeta: Dati: 0.02MB + Indice: 0.03MB
wp_woocommerce_log: Dati: 0.02MB + Indice: 0.02MB
deva_commentmeta: Dati: 43.58MB + Indice: 6.03MB
deva_comments: Dati: 45.58MB + Indice: 17.58MB
deva_fv_fp_hls_access_tokens: Dati: 0.02MB + Indice: 0.03MB
deva_links: Dati: 0.02MB + Indice: 0.02MB
deva_myCRED_log: Dati: 3.52MB + Indice: 0.20MB
deva_newsletter: Dati: 0.02MB + Indice: 0.02MB
deva_newsletter_emails: Dati: 0.02MB + Indice: 0.00MB
deva_newsletter_sent: Dati: 0.02MB + Indice: 0.03MB
deva_newsletter_stats: Dati: 0.02MB + Indice: 0.00MB
deva_options: Dati: 2.08MB + Indice: 0.03MB
deva_postmails: Dati: 0.02MB + Indice: 0.00MB
deva_postmeta: Dati: 1.52MB + Indice: 0.64MB
deva_posts: Dati: 2.52MB + Indice: 0.42MB
deva_termmeta: Dati: 0.02MB + Indice: 0.03MB
deva_terms: Dati: 0.02MB + Indice: 0.03MB
deva_term_relationships: Dati: 0.02MB + Indice: 0.02MB
deva_term_taxonomy: Dati: 0.02MB + Indice: 0.03MB
deva_ulike: Dati: 0.02MB + Indice: 0.00MB
deva_ulike_activities: Dati: 0.02MB + Indice: 0.00MB
deva_ulike_comments: Dati: 0.02MB + Indice: 0.00MB
deva_ulike_forums: Dati: 1.52MB + Indice: 0.00MB
fv_fp_hls_access_tokens: Dati: 0.02MB + Indice: 0.03MB
in_commentmeta: Dati: 0.02MB + Indice: 0.03MB
in_comments: Dati: 0.02MB + Indice: 0.08MB
in_cpm_file_relationship: Dati: 0.02MB + Indice: 0.02MB
in_cpm_project_items: Dati: 0.02MB + Indice: 0.02MB
in_cpm_tasks: Dati: 0.02MB + Indice: 0.00MB
in_cpm_user_role: Dati: 0.02MB + Indice: 0.00MB
in_erp_ac_banks: Dati: 0.02MB + Indice: 0.02MB
in_erp_ac_chart_classes: Dati: 0.02MB + Indice: 0.00MB
in_erp_ac_chart_types: Dati: 0.02MB + Indice: 0.02MB
in_erp_ac_journals: Dati: 0.02MB + Indice: 0.03MB
in_erp_ac_ledger: Dati: 0.02MB + Indice: 0.05MB
in_erp_ac_payments: Dati: 0.02MB + Indice: 0.00MB
in_erp_ac_tax: Dati: 0.02MB + Indice: 0.00MB
in_erp_ac_tax_items: Dati: 0.02MB + Indice: 0.00MB
in_erp_ac_transactions: Dati: 0.02MB + Indice: 0.06MB
in_erp_ac_transaction_items: Dati: 0.02MB + Indice: 0.05MB
in_erp_audit_log: Dati: 0.02MB + Indice: 0.06MB
in_erp_company_locations: Dati: 0.02MB + Indice: 0.02MB
in_erp_crm_activities_task: Dati: 0.02MB + Indice: 0.03MB
in_erp_crm_contact_group: Dati: 0.02MB + Indice: 0.00MB
in_erp_crm_contact_subscriber: Dati: 0.02MB + Indice: 0.05MB
in_erp_crm_customer_activities: Dati: 0.02MB + Indice: 0.06MB
in_erp_crm_customer_companies: Dati: 0.02MB + Indice: 0.03MB
in_erp_crm_save_email_replies: Dati: 0.02MB + Indice: 0.00MB
in_erp_crm_save_search: Dati: 0.02MB + Indice: 0.00MB
in_erp_hr_announcement: Dati: 0.02MB + Indice: 0.05MB
in_erp_hr_dependents: Dati: 0.02MB + Indice: 0.02MB
in_erp_hr_depts: Dati: 0.02MB + Indice: 0.00MB
in_erp_hr_designations: Dati: 0.02MB + Indice: 0.00MB
in_erp_hr_education: Dati: 0.02MB + Indice: 0.02MB
in_erp_hr_employees: Dati: 0.02MB + Indice: 0.08MB
in_erp_hr_employee_history: Dati: 0.02MB + Indice: 0.03MB
in_erp_hr_employee_notes: Dati: 0.02MB + Indice: 0.00MB
in_erp_hr_employee_performance: Dati: 0.02MB + Indice: 0.02MB
in_erp_hr_holiday: Dati: 0.02MB + Indice: 0.00MB
in_erp_hr_leaves: Dati: 0.02MB + Indice: 0.03MB
in_erp_hr_leave_entitlements: Dati: 0.02MB + Indice: 0.03MB
in_erp_hr_leave_policies: Dati: 0.02MB + Indice: 0.00MB
in_erp_hr_leave_requests: Dati: 0.02MB + Indice: 0.08MB
in_erp_hr_work_exp: Dati: 0.02MB + Indice: 0.02MB
in_erp_peoplemeta: Dati: 0.02MB + Indice: 0.03MB
in_erp_peoples: Dati: 0.02MB + Indice: 0.06MB
in_erp_people_types: Dati: 0.02MB + Indice: 0.02MB
in_erp_people_type_relations: Dati: 0.02MB + Indice: 0.03MB
in_links: Dati: 0.02MB + Indice: 0.02MB
in_options: Dati: 0.20MB + Indice: 0.02MB
in_postmeta: Dati: 0.02MB + Indice: 0.03MB
in_posts: Dati: 0.02MB + Indice: 0.06MB
in_termmeta: Dati: 0.02MB + Indice: 0.03MB
in_terms: Dati: 0.02MB + Indice: 0.03MB
in_term_relationships: Dati: 0.02MB + Indice: 0.02MB
in_term_taxonomy: Dati: 0.02MB + Indice: 0.03MB
mt_bad_behavior: Dati: 5.52MB + Indice: 0.31MB
mt_commentmeta: Dati: 17.55MB + Indice: 16.03MB
mt_country: Dati: 0.19MB + Indice: 0.00MB
mt_fv_fp_hls_access_tokens: Dati: 0.02MB + Indice: 0.03MB
mt_groupon: Dati: 0.02MB + Indice: 0.00MB
mt_links: Dati: 0.02MB + Indice: 0.03MB
mt_no_disposable_email: Dati: 0.02MB + Indice: 0.02MB
mt_options: Dati: 19.50MB + Indice: 5.39MB
mt_postmeta: Dati: 595.00MB + Indice: 493.00MB
mt_posts: Dati: 161.67MB + Indice: 65.72MB
mt_promo: Dati: 2.52MB + Indice: 0.00MB
mt_queue: Dati: 0.02MB + Indice: 0.00MB
mt_ratings: Dati: 1.52MB + Indice: 0.00MB
mt_termmeta: Dati: 0.19MB + Indice: 0.25MB
mt_terms: Dati: 0.23MB + Indice: 0.28MB
mt_term_relationships: Dati: 7.52MB + Indice: 6.52MB
mt_term_taxonomy: Dati: 0.23MB + Indice: 0.27MB
mt_woocommerce_api_keys: Dati: 0.02MB + Indice: 0.03MB
mt_woocommerce_attribute_taxonomies: Dati: 0.02MB + Indice: 0.02MB
mt_woocommerce_downloadable_product_permissions: Dati: 13.52MB + Indice: 32.77MB
mt_woocommerce_log: Dati: 0.02MB + Indice: 0.02MB
mt_woocommerce_order_itemmeta: Dati: 93.64MB + Indice: 104.30MB
mt_woocommerce_order_items: Dati: 16.55MB + Indice: 9.52MB
mt_woocommerce_payment_tokenmeta: Dati: 0.02MB + Indice: 0.03MB
mt_woocommerce_payment_tokens: Dati: 0.02MB + Indice: 0.02MB
mt_woocommerce_sessions: Dati: 0.42MB + Indice: 0.05MB
mt_woocommerce_shipping_zones: Dati: 0.02MB + Indice: 0.00MB
mt_woocommerce_shipping_zone_locations: Dati: 0.02MB + Indice: 0.05MB
mt_woocommerce_shipping_zone_methods: Dati: 0.02MB + Indice: 0.00MB
mt_woocommerce_tax_rates: Dati: 0.02MB + Indice: 0.06MB
mt_woocommerce_tax_rate_locations: Dati: 0.02MB + Indice: 0.05MB
mt_woocommerce_termmeta: Dati: 0.02MB + Indice: 0.03MB
mt_wp_comments: Dati: 149.69MB + Indice: 130.91MB
mt_yith_wcaf_affiliatemeta: Dati: 7.52MB + Indice: 0.69MB
mt_yith_wcaf_affiliates: Dati: 0.11MB + Indice: 0.00MB
mt_yith_wcaf_clicks: Dati: 146.66MB + Indice: 0.00MB
mt_yith_wcaf_commissions: Dati: 2.52MB + Indice: 0.00MB
mt_yith_wcaf_commission_notes: Dati: 0.02MB + Indice: 0.00MB
mt_yith_wcaf_payments: Dati: 0.02MB + Indice: 0.00MB
mt_yith_wcaf_payment_commission: Dati: 1.52MB + Indice: 0.00MB
mt_yith_wcaf_payment_notes: Dati: 0.02MB + Indice: 0.00MB
mt_yith_wcaf_plan: Dati: 0.23MB + Indice: 0.00MB
mt_yith_wcaf_rates: Dati: 0.02MB + Indice: 0.00MB
ps_commentmeta: Dati: 0.27MB + Indice: 0.14MB
ps_comments: Dati: 1.50MB + Indice: 0.33MB
ps_links: Dati: 0.02MB + Indice: 0.02MB
ps_my_calendar: Dati: 0.02MB + Indice: 0.02MB
ps_my_calendar_categories: Dati: 0.02MB + Indice: 0.00MB
ps_my_calendar_category_relationships: Dati: 0.02MB + Indice: 0.02MB
ps_my_calendar_events: Dati: 0.02MB + Indice: 0.02MB
ps_my_calendar_locations: Dati: 0.02MB + Indice: 0.00MB
ps_options: Dati: 1.06MB + Indice: 0.02MB
ps_postmeta: Dati: 0.22MB + Indice: 0.22MB
ps_posts: Dati: 0.27MB + Indice: 0.23MB
ps_termmeta: Dati: 0.02MB + Indice: 0.03MB
ps_terms: Dati: 0.02MB + Indice: 0.03MB
ps_term_relationships: Dati: 0.02MB + Indice: 0.02MB
ps_term_taxonomy: Dati: 0.02MB + Indice: 0.03MB
wpdev_usermeta: Dati: 0.08MB + Indice: 0.03MB
wpdev_users: Dati: 0.02MB + Indice: 0.05MB
wp_bad_behavior: Dati: 1.09MB + Indice: 0.03MB
wp_commentmeta: Dati: 4.52MB + Indice: 5.03MB
wp_comments: Dati: 35.56MB + Indice: 36.09MB
wp_country: Dati: 0.19MB + Indice: 0.00MB
wp_cpm_file_relationship: Dati: 0.02MB + Indice: 0.02MB
wp_cpm_project_items: Dati: 0.02MB + Indice: 0.02MB
wp_cpm_tasks: Dati: 0.02MB + Indice: 0.00MB
wp_cpm_user_role: Dati: 0.02MB + Indice: 0.00MB
wp_erp_ac_banks: Dati: 0.02MB + Indice: 0.02MB
wp_erp_ac_chart_classes: Dati: 0.02MB + Indice: 0.00MB
wp_erp_ac_chart_types: Dati: 0.02MB + Indice: 0.02MB
wp_erp_ac_journals: Dati: 0.02MB + Indice: 0.03MB
wp_erp_ac_ledger: Dati: 0.02MB + Indice: 0.05MB
wp_erp_ac_payments: Dati: 0.02MB + Indice: 0.00MB
wp_erp_ac_tax: Dati: 0.02MB + Indice: 0.00MB
wp_erp_ac_tax_items: Dati: 0.02MB + Indice: 0.00MB
wp_erp_ac_transactions: Dati: 0.02MB + Indice: 0.06MB
wp_erp_ac_transaction_items: Dati: 0.02MB + Indice: 0.05MB
wp_erp_audit_log: Dati: 0.02MB + Indice: 0.06MB
wp_erp_company_locations: Dati: 0.02MB + Indice: 0.02MB
wp_erp_crm_activities_task: Dati: 0.02MB + Indice: 0.03MB
wp_erp_crm_contact_group: Dati: 0.02MB + Indice: 0.00MB
wp_erp_crm_contact_subscriber: Dati: 0.02MB + Indice: 0.05MB
wp_erp_crm_customer_activities: Dati: 0.02MB + Indice: 0.06MB
wp_erp_crm_customer_companies: Dati: 0.02MB + Indice: 0.03MB
wp_erp_crm_save_email_replies: Dati: 0.02MB + Indice: 0.00MB
wp_erp_crm_save_search: Dati: 0.02MB + Indice: 0.00MB
wp_erp_hr_announcement: Dati: 0.02MB + Indice: 0.05MB
wp_erp_hr_dependents: Dati: 0.02MB + Indice: 0.02MB
wp_erp_hr_depts: Dati: 0.02MB + Indice: 0.00MB
wp_erp_hr_designations: Dati: 0.02MB + Indice: 0.00MB
wp_erp_hr_education: Dati: 0.02MB + Indice: 0.02MB
wp_erp_hr_employees: Dati: 0.02MB + Indice: 0.08MB
wp_erp_hr_employee_history: Dati: 0.02MB + Indice: 0.03MB
wp_erp_hr_employee_notes: Dati: 0.02MB + Indice: 0.00MB
wp_erp_hr_employee_performance: Dati: 0.02MB + Indice: 0.02MB
wp_erp_hr_holiday: Dati: 0.02MB + Indice: 0.00MB
wp_erp_hr_leaves: Dati: 0.02MB + Indice: 0.03MB
wp_erp_hr_leave_entitlements: Dati: 0.02MB + Indice: 0.03MB
wp_erp_hr_leave_policies: Dati: 0.02MB + Indice: 0.00MB
wp_erp_hr_leave_requests: Dati: 0.02MB + Indice: 0.08MB
wp_erp_hr_work_exp: Dati: 0.02MB + Indice: 0.02MB
wp_erp_peoplemeta: Dati: 0.02MB + Indice: 0.03MB
wp_erp_peoples: Dati: 0.02MB + Indice: 0.06MB
wp_erp_people_types: Dati: 0.02MB + Indice: 0.02MB
wp_erp_people_type_relations: Dati: 0.02MB + Indice: 0.03MB
wp_fv_fp_hls_access_tokens: Dati: 0.02MB + Indice: 0.03MB
wp_fv_player_playermeta: Dati: 0.02MB + Indice: 0.03MB
wp_fv_player_players: Dati: 0.02MB + Indice: 0.00MB
wp_fv_player_videometa: Dati: 0.02MB + Indice: 0.03MB
wp_fv_player_videos: Dati: 0.02MB + Indice: 0.03MB
wp_links: Dati: 0.02MB + Indice: 0.03MB
wp_no_disposable_email: Dati: 0.02MB + Indice: 0.02MB
wp_options: Dati: 4.20MB + Indice: 0.08MB
wp_pcp_leads: Dati: 0.06MB + Indice: 0.02MB
wp_postmeta: Dati: 34.56MB + Indice: 19.06MB
wp_posts: Dati: 69.58MB + Indice: 8.47MB
wp_tcb_api_error_log: Dati: 0.02MB + Indice: 0.00MB
wp_termmeta: Dati: 0.02MB + Indice: 0.03MB
wp_terms: Dati: 0.22MB + Indice: 0.28MB
wp_term_relationships: Dati: 7.52MB + Indice: 6.52MB
wp_term_taxonomy: Dati: 0.23MB + Indice: 0.23MB
wp_tge_answers: Dati: 0.02MB + Indice: 0.02MB
wp_tge_questions: Dati: 0.02MB + Indice: 0.02MB
wp_tho_event_log: Dati: 0.02MB + Indice: 0.00MB
wp_tho_tests: Dati: 0.02MB + Indice: 0.00MB
wp_tho_test_items: Dati: 0.02MB + Indice: 0.00MB
wp_tqb_event_log: Dati: 0.02MB + Indice: 0.00MB
wp_tqb_results: Dati: 0.02MB + Indice: 0.00MB
wp_tqb_tests: Dati: 0.02MB + Indice: 0.00MB
wp_tqb_tests_items: Dati: 0.02MB + Indice: 0.00MB
wp_tqb_users: Dati: 0.02MB + Indice: 0.00MB
wp_tqb_user_answers: Dati: 0.02MB + Indice: 0.00MB
wp_tqb_variations: Dati: 0.02MB + Indice: 0.00MB
wp_tve_leads_contacts: Dati: 0.02MB + Indice: 0.00MB
wp_tve_leads_contact_download: Dati: 0.02MB + Indice: 0.00MB
wp_tve_leads_event_log: Dati: 0.02MB + Indice: 0.00MB
wp_tve_leads_form_variations: Dati: 0.02MB + Indice: 0.06MB
wp_tve_leads_group_options: Dati: 0.02MB + Indice: 0.00MB
wp_tve_leads_saved_group_options: Dati: 0.02MB + Indice: 0.00MB
wp_tve_leads_split_test: Dati: 0.02MB + Indice: 0.00MB
wp_tve_leads_split_test_items: Dati: 0.02MB + Indice: 0.00MB
wp_tve_ult_designs: Dati: 0.02MB + Indice: 0.00MB
wp_tve_ult_emails: Dati: 0.02MB + Indice: 0.00MB
wp_tve_ult_events: Dati: 0.02MB + Indice: 0.00MB
wp_tve_ult_event_log: Dati: 0.02MB + Indice: 0.00MB
wp_tve_ult_settings_campaign: Dati: 0.02MB + Indice: 0.00MB
wp_tve_ult_settings_templates: Dati: 0.02MB + Indice: 0.00MB
wp_usermeta: Dati: 152.70MB + Indice: 158.45MB
wp_users: Dati: 15.52MB + Indice: 13.55MB
wp_wcpdf_credit_note_number: Dati: 0.02MB + Indice: 0.00MB
wp_wcpdf_invoice_number: Dati: 0.02MB + Indice: 0.00MB
wp_wcpdf_proforma_number: Dati: 0.02MB + Indice: 0.00MB
wp_wc_download_log: Dati: 0.02MB + Indice: 0.03MB
wp_wc_webhooks: Dati: 0.02MB + Indice: 0.02MB
wp_woocommerce_termmeta: Dati: 0.02MB + Indice: 0.03MB
wp_wplc_chat_msgs: Dati: 0.02MB + Indice: 0.00MB
wp_wplc_chat_sessions: Dati: 0.02MB + Indice: 0.00MB
wp_wplc_offline_messages: Dati: 0.02MB + Indice: 0.00MB
wp_wplc_webhooks: Dati: 0.02MB + Indice: 0.00MB
wp_yith_wcaf_affiliatemeta: Dati: 0.02MB + Indice: 0.03MB
wp_yith_wcaf_affiliates: Dati: 0.02MB + Indice: 0.00MB
wp_yith_wcaf_clicks: Dati: 0.02MB + Indice: 0.00MB
wp_yith_wcaf_commissions: Dati: 0.02MB + Indice: 0.00MB
wp_yith_wcaf_commission_notes: Dati: 0.02MB + Indice: 0.00MB
wp_yith_wcaf_payments: Dati: 0.02MB + Indice: 0.00MB
wp_yith_wcaf_payment_commission: Dati: 0.02MB + Indice: 0.00MB
wp_yith_wcaf_payment_notes: Dati: 0.02MB + Indice: 0.00MB
wp_yith_wcaf_rates: Dati: 0.02MB + Indice: 0.00MB
wp_yoast_seo_links: Dati: 0.09MB + Indice: 0.05MB
wp_yoast_seo_meta: Dati: 0.02MB + Indice: 0.00MB

### Post Type Counts ###

attachment: 7378
cookielawinfo: 1
course: 8
course_features: 45
course_promo: 3
cpm_project: 1
csshortcode: 101
dsn_note: 1
erp_hr_announcement: 1
lesson: 204
nav_menu_item: 92
newsletter: 8
oembed_cache: 1
page: 1184
pcplms_upsell: 7
pcplms_upsell_roles: 5
pcp_survey: 6288
post: 3434
product: 868
revision: 810
shop_coupon: 18
shop_order: 1597
shop_order_refund: 1323
smart_offers: 89
snp_popups: 1
testimonial: 9
thrive_image: 1
tqb_quiz: 1
tqb_results: 1
tve_form_type: 3
tve_lead_group: 1
wysijap: 1

### Security ###

Secure connection (HTTPS): ❌
					Il tuo negozio non utilizza HTTPS. Scopri di più su HTTPS e certificati SSL .
Hide errors from visitors: ✔

### Active Plugins (36) ###

Query Monitor: by John Blackbourn & contributors – 3.2.2
PCP Salvataggio dati questionari: by david silvestri@dot4all S.r.l. – 0.0.2
User Switching in Admin Bar: by Mark Wilkinson – 1.1.1
Advanced Cron Manager: by BracketSpace – 2.3.6
Akismet Anti-Spam: by Automattic – 4.1.1
Bad Behavior: by Michael Hampton – 2.2.22
BE Subpages Widget: by Bill Erickson – 1.7
Better Click To Tweet: by Ben Meredith – 5.7.2
Classic Editor: by Contributori WordPress – 1.4
CMS Tree Page View: by Pär Thernström – 1.6
GDPR Cookie Consent: by WebToffee – 1.7.3
Disable admin notification email: by David Silvestri – 0.0.1
Enhanced Media Library: by wpUXsolutions – 2.7.2
ezPHP: by WebSharks
Inc. (Jason Caldwell) – 160128

Featured Image by URL: by Knawat Team – 1.1.3
FV Player Pro: by Foliovision – 7.3.9.727
FV Player: by  – 7.3.7.727
Prevent Concurrent Logins: by Frankie Jarrett – 0.4.0
Rewrite Rules Inspector: by Daniel Bachhuber
Automattic – 1.2.1

Send E-mails with Mandrill: by Miller Media ( Matt Miller ) – 1.1
UberMenu Conditionals: by Chris Mavricos
SevenSpark – 3.0

UberMenu 3 - The Ultimate WordPress Mega Menu: by Chris Mavricos
SevenSpark – 3.4.1.1

User Role Editor: by Vladimir Garagulya – 4.49
User Switching: by John Blackbourn e collaboratori – 1.4.1
WooCommerce PayPal Powered by Braintree Gateway: by WooCommerce – 2.2.0
Advanced Order Export For WooCommerce (Pro): by AlgolPlus – 1.5.2 – Non testato con la versione attiva di WooCommerce
WooCommerce PDF Invoices & Packing Slips: by Ewout Fernhout – 2.2.6
WooCommerce PDF Invoices & Packing Slips Professional: by Ewout Fernhout – 2.2.11
WooCommerce Store Credit: by WooThemes – 2.1.5 – 2.3.0 è disponibile – Non testato con la versione attiva di WooCommerce
WooCommerce: by Automattic – 3.5.4
Yoast SEO: by Team Yoast – 9.5
WP Overnight Sidekick: by Jeremiah Prummer
Ewout Fernhout
Michael Kluver – 2.0.4

WP Performance Tester: by <a href="http://reviewsignal.com">Kevin Ohashi</a> – 0.1
PcP Affiliates: by david.silvestri –
YITH WooCommerce Product Bundles Premium: by YITHEMES – 1.2.10 – Non testato con la versione attiva di WooCommerce
YITH WooCommerce Waiting List: by YITH – 1.3.4

### Settings ###

API Enabled: ✔
Force SSL: ✔
Currency: EUR (€)
Currency Position: right
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)
woosb (woosb)
yith_bundle (yith_bundle)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Shop base: ❌ L'ID della pagina è impostato
ma la pagina non esiste

Carrello: #47192 - /carrello/
Cassa: #15135 - /checkout/
Il mio account: #15148 - /account/
Termini e condizioni: #66032 - /termini/

### Theme ###

Name: PcP
Version: – 0 è disponibile
Author URL: 
Child Theme: ❌ – Se stai modificando WooCommerce o un tema genitore che non hai costruito personalmente
ti consigliamo di utilizzare un child theme. Vedi: Come creare un child theme

WooCommerce Support: ✔

### Templates ###

Archive Template: Il tuo tema ha il file woocommerce.php
non potrai sovrascrivere il template personalizzato woocommerce/archive-product.php poichè woocommerce.php ha priorità su archive-product.php. Questo serve a prevenire problemi di visualizzazione.

Overrides: qf/woocommerce/checkout/cart-errors.php
qf/woocommerce/checkout/form-checkout.php
qf/woocommerce/checkout/form-shipping.php
qf/woocommerce/checkout/payment-method.php
qf/woocommerce/checkout/payment.php
qf/woocommerce/checkout/review-order.php
qf/woocommerce/checkout/thankyou.php
qf/woocommerce/content-product.php
qf/woocommerce/emails/admin-failed-order.php
qf/woocommerce/emails/customer-completed-order.php
qf/woocommerce/emails/customer-processing-order.php
qf/woocommerce/emails/customer-refunded-order.php
qf/woocommerce/emails/email-downloads.php
qf/woocommerce/myaccount/dashboard.php
qf/woocommerce/myaccount/form-edit-account.php
qf/woocommerce/myaccount/form-edit-address.php
qf/woocommerce/myaccount/form-login.php
qf/woocommerce/myaccount/form-lost-password.php
qf/woocommerce/myaccount/my-orders.php
qf/woocommerce/myaccount/view-order.php


### Braintree (Credit Card) ###

Environment: Sandbox
Tokenization Enabled: –
Debug Mode: Off

### Braintree (PayPal) ###

Environment: Sandbox
Tokenization Enabled: –
Debug Mode: Save to Log
`
```
</details>

**Comment by mikejolley** [comment]
Hi,

You'll need to filter the country locale too (or instead of) billing fields. This is because some countries need a different display order of fields. If we just used `woocommerce_billing_fields`, all countries would be forced to use the same order.

> This happen only with WC 3.5.4 and not previous version 3.5.3. I've production server with WC 3.5.3 and stage server with 3.5.4.

This is because the locale update on checkout first load was broken in .3 and was fixed in .4. If you'd switched countries in .3 and back again, you'd see a different order used than your custom order in `woocommerce_billing_fields`

Hope that makes sense.

Thanks
